docs/contribute.md

```diff
@@ -20,63 +20,36 @@ Prepare your system by installing prerequisites.
 
 You need to install system libraries, as described in {ref}`plone:plone-prerequisites-label`, with the exception of GNU make.
 
-### tox
-
-[tox](https://tox.wiki/en/stable/index.html) automates and standardizes testing in Python.
-Install tox into your Python user space with the following command.
-
-```shell
-python -m pip install --user tox
-```
-
-### pre-commit
```

Comment:
Same. How do we use pre-commit if it is not installed? Does tox handle it as a dependency?

Reply:
All make commands that run tox should depend on $(BIN_FOLDER)/tox which installs tox etc and activates the pre-commit hook. See https://github.com/plone/plone.api/blob/docs-enhancements/Makefile#L48-L52 , a wonderful trick, lend from plone.meta

Reply:
Nice!